### DISCUSSION TITLE
Stylus pressure recognision is supported by Wayland's remote desktop or wayvnc?

### DISCUSSION THREAD

**Discussion by Inqb8tr-jp** [any1/wayvnc#155] [General]
Hi.
I recently bought Lenovo Ideapad Duet 5 Chromebook to use as a drawing tablet.
But basically as I am Linux fan, I want to work with Linux apps like Krita or Gimp. 
So I install Linux app on Crostini Debian Linux on ChromeOS.
But it lacks the stylus pressure recognition on Linux apps on ChromeOS by now.

As a alternative method, now I testing Wayland and wayvnc. 
I successfully set Manjaro Sway edition up.
But tigervnc's client, vncviwer, don't have ability to recognize pen pressure.

I don't know that here is the right place to ask but I want to ask if Wayland or X11's remote desktop have a support of stylus pressure recognition?
If here is not the right place to ask, where should I ask to?

**Comment by any1** [comment]
I can tell you that wayvnc does not have any direct support for drawing tablets and I do not know of any VNC protocol extension for drawing tablets.

There is support for drawing tablets in sway, and if I remember correctly, it supports pressure.

The last time I tried Krita though, it didn't have native Wayland support, but that was almost 2 years ago.

**Reply by Inqb8tr-jp** [reply]
Thanks for your reply.
I've further studied ChromeOS's Sommelier is also a Wayland compositor. 
https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/vm_tools/sommelier/README.md

Lack of recognition for stylus pressure is  probably based on the same, Wayland.
I'll ask Wayland community. 
Thanks.
